driver/reduceextreme_driver.hpp

```diff
@@ -0,0 +1,509 @@
+    }
+
+    return miopenStatusSuccess;
+}
+
+template <typename Tgpu, typename Tref>
+int ReduceExtremeDriver<Tgpu, Tref>::RunForwardGPU()
+{
+    float kernel_total_time = 0;
+    float kernel_first_time = 0;
+
+    Timer t;
+    START_TIME
+
+    for(int32_t i = 0; i < inflags.GetValueInt("iter"); i++)
+    {
+        if((reduceExtremeOp == MIOPEN_REDUCE_EXTREME_MIN) ||
+           (reduceExtremeOp == MIOPEN_REDUCE_EXTREME_MAX))
+        {
+            miopenReduceExtremeForward(GetHandle(),
+                                       xDesc,
+                                       x_dev->GetMem(),
+                                       dim,
+                                       reduceExtremeOp,
+                                       yDesc,
+                                       y_dev->GetMem(),
+                                       indiceDesc,
+                                       indice_dev->GetMem());
+        }
+        else
+        {
+            miopenReduceExtremeForward(GetHandle(),
+                                       xDesc,
+                                       x_dev->GetMem(),
+                                       dim,
+                                       reduceExtremeOp,
+                                       nullptr,
+                                       nullptr,
+                                       indiceDesc,
+                                       indice_dev->GetMem());
+        }
+
+        float time = 0;
+        miopenGetKernelTime(GetHandle(), &time);
+        kernel_total_time += time;
+        if(i == 0)
+            kernel_first_time = time;
+    }
+
+    if(inflags.GetValueInt("time") == 1)
+    {
+        STOP_TIME
+        int32_t iter = inflags.GetValueInt("iter");
+        if(WALL_CLOCK)
+            std::cout << "Wall-clock Time Forward ReduceExtreme Elapsed: " << t.gettime_ms() / iter
+                      << " ms\n";
+
+        float kernel_average_time =
+            iter > 1 ? (kernel_total_time - kernel_first_time) / (iter - 1) : kernel_first_time;
+        std::cout << "GPU Kernel Time Forward ReduceExtreme Elapsed: " << kernel_average_time
+                  << " ms\n";
+    }
+
+    if(indice_dev->FromGPU(GetStream(), indice.data()) != 0)
+        std::cerr << "Error copying (indice_dev) from GPU, size: " << indice_dev->GetSize()
+                  << std::endl;
+
+    if((reduceExtremeOp == MIOPEN_REDUCE_EXTREME_MIN) ||
+       (reduceExtremeOp == MIOPEN_REDUCE_EXTREME_MAX))
+    {
+        if(y_dev->FromGPU(GetStream(), y.data()) != 0)
+            std::cerr << "Error copying (y_dev) from GPU, size: " << y_dev->GetSize() << std::endl;
+    }
+
+    return miopenStatusSuccess;
+}
+
+template <typename Tgpu, typename Tref>
+int ReduceExtremeDriver<Tgpu, Tref>::RunForwardCPU()
+{
+    if(reduceExtremeOp == MIOPEN_REDUCE_EXTREME_ARGMIN)
+    {
+        mloReduceExtremeForwardRunHost<Tgpu, Tref, ReduceExtremeOp_t::Argmin>(
```

Comment:
[may be postponed] Here and below
```suggestion
        return mloReduceExtremeForwardRunHost<Tgpu, Tref, ReduceExtremeOp_t::Argmin>(
```

Reply:
fixed it.

Reply:
[resolved]